server/handlers/default.go

```diff
@@ -176,15 +187,18 @@ func GetKeyHandler(ctx context.Context, w http.ResponseWriter, r *http.Request)
 func getKeyHandler(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
 	gun, ok := vars["imageName"]
 	if !ok || gun == "" {
+		logrus.Info("400 GET no gun in request")
```

Comment:
I don't think we can for this one because we don't have the gun yet, but totally for the other `logrus.Error` (and I think I see a `logrus.Warnf` that can get fixed)